lib/test_env.rb

```diff
@@ -0,0 +1,13 @@
+def setup_test_env
+  _disable_animations
+end
+
+def _disable_animations
```

Comment:
@abhinavmsra I think we don't use this convention (I mean underscore before the method name)